pkg/api/accesscontrol.go

```diff
@@ -128,8 +128,8 @@ func (hs *HTTPServer) declareFixedRoles() error {
 			Group:       "Infrequently used",
 			Permissions: []ac.Permission{
 				{
-					Action: datasources.ActionDatasourcesIDRead,
-					Scope:  datasources.ScopeDatasourcesAll,
+					Action: datasources.ActionIDRead,
```

Comment:
nit: not for this PR, but I think we can simplify this and not have an additional action for reading data source ID.

Reply:
Yes i agree